tests/unit/Jobs/Emails/PrepareMotionSummaryTest.php

```diff
@@ -100,7 +100,7 @@ public function motion_summary_email_contains_motions_that_recently_closed()
 
         dispatch(new PrepareMotionSummary());
 
-        Mail::assertSentTo([$user], MotionSummary::class, function ($mail) use ($motion) {
+        Mail::assertSent(MotionSummary::class, function ($mail) use ($motion) {
```

Comment:
The replacement for "assertSentTo" is not "assertSent", you also need to now add a "to" check like the documentation.